src/output.rs

```diff
@@ -59,7 +59,20 @@ impl Output {
         for filter in &self.filters {
             let f = try!(context.get_filter(&filter.name)
                 .ok_or(Error::Render(format!("Filter {} not implemented", &filter.name))));
-            entry = try!(f(&entry, &filter.arguments));
+
+            let mut arguments = Vec::new();
+            for arg in filter.arguments.iter() {
```

Comment:
Was getting errors when I removed the iter. Changed it to loop over `&filter.arguments` instead.